src/storage/page_buffer.c

```diff
@@ -7914,6 +7941,11 @@ pgbuf_victimize_bcb (THREAD_ENTRY * thread_p, PGBUF_BCB * bufptr)
     }
   assert (bufptr->latch_mode == PGBUF_NO_LATCH);
 
+  if (pgbuf_Pool.get_highest_evicted_lsa () < bufptr->iopage_buffer->iopage.prv.lsa)
+    {
+      pgbuf_Pool.update_highest_evicted_lsa (bufptr->iopage_buffer->iopage.prv.lsa);
+    }
```

Comment:
pgbuf_Pool.update_highest_evicted_lsa already compares & earlies out if its highest evicted LSA is higher. Avoid comparing twice.

```suggestion
  pgbuf_Pool.update_highest_evicted_lsa (bufptr->iopage_buffer->iopage.prv.lsa);
```